kubetest/OWNERS

```diff
@@ -1,14 +1,15 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-- amwat
+- BenTheElder
 - wojtek-t # For European timezone emergencies.
 - feiskyer
 approvers:
-- amwat
+- BenTheElder
```

Comment:
I'm not sure the others are active and I've worked on / reviewed / approved this many times before just from OWNERS/ rather than kubetest/OWNERS. I'm doing a lot, but for now I've agreed to inherit this along with kubetest2 (which I'm already listed in).